postgress.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+from constants import DB_NAME, DB_PASSWORD, DB_USER, HOST, PORT
+def pgconn(query):
+    if "select" in query:
+        conn = psycopg2.connect(
+            database=DB_NAME,
+            user=DB_USER,
+            password=DB_PASSWORD,
+            host=HOST,
+            port=PORT
+        )
+        try:
+            cur = conn.cursor()
+            cur.execute(query)
+            results = cur.fetchall()
+            cur.close()
+            conn.close()
+            return results
+        except Exception as e :
+            print(e)
```

Comment:
This is very, very rarely what you'd want to do here.  If you absolutely must do this, at least include the full context:

```suggestion
            traceback.print_exc(file=sys.stdout)
```